Samples/VirtualCamera/README.md

```diff
@@ -0,0 +1,93 @@
+
+### Project Setup
+
+1. Install Visual studio MSI project extension [link](https://marketplace.visualstudio.com/items?itemName=visualstudioclient.MicrosoftVisualStudio2017InstallerProjects>)
+
+2. Create MSI project in Visual studio 
+
+3. Add binaries
+This should include mediasource dll and the executable that is responsbile for the uninstallation of virtual camera.
+
+    1. Right-click on the "Setup Project", go on "Add" then click "Assembly".
+
+    2. Browse for your COM assembly (.dll) and add it to the "Setup Project".
+
+    3. After adding the assembly, right-click on it in "Solution Explorer" and click "Properties".
+
+    4. Look for "Register" property and set it to "vsdrfDoNoRegister".
+
+4. Add registry - register dll during installation
+    1. Right-click on the Project, select "View" then "Registry"
+    2. Add these keys and values
+    HKLM\software\classes\CLSID\<CLSID_GUID>
+    HKLM\software\classes\CLSID\<CLSID_GUID>\InProcServer32
+    HKLM\software\classes\CLSID\<CLSID_GUID>\InProcServer32\Default    value:[TARGETDIR]<MediaSourceDll>
+    HKLM\software\classes\CLSID\<CLSID_GUID>\InProcServer32\Threading  value:"Both"
+
+5. Add UnInstall routine
+    1. Right-click on the Project, select "View" then "Custom Actions"
+    2. Right-click on "UnInstall", select "Add Custom Action"
+    3. From the popup windows select the executable with the uninstall routine.  In this example it is VirtualCamera_ConsoleApp.exe /uninstall
+
+## VirtualCamera_AppMSIX
+----
+This project package the desktop VirtualCamera_ConsoleApp executable into a UWP application.
+This is a sample configuration application that is responsible of create / remove / configure the virtual camera related to the mediasource install via VirtualCamera_AppMSI.
+
+Altnertatively you can create a UWP application 
+
+## VirtualCameraTest
+----
+Basic unit test run VirutalCameraWin32 media soource implemented with Google test framework.  The test provides basic validation to mediasource implementation. <br> <br>
+Both HWMediaSourceUT and SimpleMediaSourceUT unit tests are used to validate the implementation of the media source is confined with guideline [here] (https://docs.microsoft.com/en-us/windows-hardware/drivers/stream/frame-server-custom-media-source#custom-media-source-dll).  These tests will load the media source in the test process ensure the basic functionality of the media source. <br> <br>
+VirtualCameraUnit required installation will create a virtual camera.  The media source in this case will be loaded in FrameServer process.  To debub the media source running as virtual camera you will need to attach debugger to FrameServer process.
+
+### How to run test
+1. Build VirtualCamera_AppMSI
+2. Install the application

```

Comment:
both build and install doesn't require admin mode. 
Install virtual camera does because I add custom property, but the application will return error message letting user know the application need to run in admin.